src/utils/error.ts

```diff
@@ -178,3 +178,18 @@ export abstract class StorybookSvelteCSFError extends Error {
     return `<Story name="${this.storyNameFromAttribute}" />`;
   }
 }
+
+// WARN: We can't use instance of `StorybookSvelteCSFError`, because is an _abstract_ class :sob:
+export function isStorybookSvelteCSFError(error: unknown) {
+  if (typeof error !== 'object' || error === null) {
+    return false;
+  }
+
+  for (const key of ['category', 'code', 'data', 'documentation', 'fullErrorCode', 'template']) {
```

Comment:
this looks like a list we'll forget to update when we change the `StorybookError` fields later, so I see two alternatives:

1. just make it minimal, like only looking for `fullErrorCode`
2. Add an explicit field to `StorybookError` like `isStorybookError = true`. Then you also wouldn't need this utility at all, you could maybe just check that field directly when needed.

Reply:
I haven't thought of this, thanks!

This definitely simplifies the process and code: 3b5e794c033191938818a667bcbec6eb3f3a6092